src/reselect.d.ts

```diff
@@ -5,22 +5,22 @@
 
 declare module Reselect {
 
-    type Selector<TInput, TOutput> = (state: TInput, props?: any) => TOutput;
+    type Selector<TInput, TProps, TOutput> = (state: TInput, TProps, props?: TProps) => TOutput;
```

Comment:
Was this a typo? That looks like you accidentally included a required implicit any parameter named TProps.